pinot-plugins/pinot-input-format/pinot-csv/src/main/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReader.java

```diff
@@ -150,6 +151,10 @@ private void init()
     _iterator = _parser.iterator();
   }
 
+  public Map<String, Integer> getCSVHeaderMap() {
+    return _parser.getHeaderMap();
```

Comment:
what happens to the record reader / headermap 

if there's no header present? we can cover this in the comment?

Reply:
added comments